### PR TITLE
Cleanup SSR

### PR DESCRIPTION
**Pull request by mmahalwy**
### Title of change
_why the change and what has changed_

### Checklist

- [ ] Unit tests written
- [ ] Manually tested
- [ ] Prettier & ESLint were run
- [ ] New dependencies are included in `package-lock.json`

### Screenshot
_insert screenshot if needed_

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32774

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32777

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32779

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32780

File: src/components/Verse/index.js [hunk 1]
```diff
@@ -16,14 +16,14 @@ import debug from 'helpers/debug';
 
 import { loadTafsirs } from '../../redux/actions/media';
 
-const Copy = Loadable({
-  loader: () => import(/* webpackChunkName: "Copy" */ 'components/Copy'),
-  LoadingComponent: ComponentLoader
+const Copy = asyncComponent({
```
Comment (sabeurthabti) [inline src/components/Verse/index.js]: just curious what the reason behind this change?

Comment (mmahalwy) [reply src/components/Verse/index.js]: `Loadable` vs  `asyncComponent`? It's cause we have 2-3 different libraries for async components and I'd like to get the bundlesize down. `async-component` felt like it was more popular than `loadable-components` and easier to work with on the server than `Loadable`

**Pull request merged**